Dockerfile

```diff
@@ -38,7 +41,11 @@ RUN apt-get update -q && apt-get upgrade -q \
     libffi-dev \
     libssl-dev \
     vim-nox
```

Comment:
Could you add a `rm` after a `update` like as https://github.com/nginxinc/docker-nginx/blob/aecb18605af6a33c1968c51df44e01e5f5a821e6/Dockerfile#L11?